set threshold to > 0 for

peterdudfield opened the pull request:
# Pull Request

## Description

Simple change to filter on GSP threshold mw > 0

Fixes #303 

## How Has This Been Tested?

normal unittests

- [ ] No
- [ ] Yes

## Checklist:

- [ ] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked my code and corrected any misspellings

JackKelly reviewed (approved):
LGTM!  Good work!  You're really powering through lots of issues!  Very exciting to see!

One tiny comment:  Maybe change the comment on line 96 of `gsp_data_source.py` so it says something like "drop any GSP below threshold_mw."  (i.e. remove the "20 MW").

I took the liberty of very slightly editing the PR description to change "Fixes #303" to "Related to issues #303 and #205" because, in 2022, it might still be nice to implement a config option so users can set the GSP threshold_mw.

peterdudfield commented:
> LGTM! Good work! You're really powering through lots of issues! Very exciting to see!
> 
> One tiny comment: Maybe change the comment on line 96 of `gsp_data_source.py` so it says something like "drop any GSP below threshold_mw." (i.e. remove the "20 MW").
> 
> I took the liberty of very slightly editing the PR description: I changed "Fixes #303" to "Related to issues #303 and #205" because, in 2022, it might still be nice to implement a config option so users can set the GSP threshold_mw. And I moved issue #303 from WP1 to WP2

Thank you
